projects/Mallard/src/components/styled-text.tsx

```diff
@@ -1,30 +1,64 @@
 import React from 'react'
 import { Text, View } from 'react-native'
-import { color } from '../theme/color'
+import { useAppAppearance } from '../theme/appearance'
 import { metrics } from '../theme/spacing'
+import { color } from '../theme/color'
 
-export class HeadlineText extends React.Component<{ style?: any }> {
-    render() {
-        return (
-            <Text
-                {...this.props}
-                style={{
-                    fontFamily: 'GHGuardianHeadline-Bold',
+export const HeadlineText = ({
+    style,
+    ...props
+}: {
+    children: string
+    style?: {}
+}) => {
+    return (
+        <Text
+            {...props}
+            style={[
+                {
+                    fontFamily: 'GHGuardianHeadline-Medium',
                     fontSize: 24,
-                }}
-            />
-        )
-    }
+                    color: color.text,
```

Comment:
Would we want to use `useAppAppearance()` here instead of `color`? I wonder if we can codify the conditions under which we would ignore the theme and apply the default colour directly?

Reply:
could be done, the reason it's not being done is that headlines are a bit different (content) from, the accessory UI and currently no parts of the themable UI display headlines. headlines do consume the article from context and the way they ingest that is by spreading a style prop on them. Not the best but wanna see how the design develops before worrying too much about this. This works for the current design and when it doesn't work anymore we can revisit!